snuba/api.py

```diff
@@ -66,6 +68,32 @@ def check_clickhouse():
 )
 
 
+@application.errorhandler(BadRequest)
+def handle_bad_request(exception: BadRequest):
+    cause = getattr(exception, '__cause__', None)
+    if cause is not None and isinstance(cause, json.errors.JSONDecodeError):
```

Comment:
Is this the same as:
```
if isinstance(cause, json.errors.JSONDecodeError):
```
?

Reply:
Yeah — good point, `NoneType` is still a type so this doesn't need to be guarded by the `cause is not None` condition. I'll clean this up, thanks!

Reply:
Improved this with 3d8d37432be3df5e735b9db7d593e54c3ddaff64.